src/applications/disability-benefits/all-claims/tests/components/UpdateMilitaryHistory.unit.spec.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { expect } from 'chai';
+import { mount } from 'enzyme';
+
+import { SAVED_SEPARATION_DATE } from '../../constants';
+import { UpdateMilitaryHistory } from '../../components/UpdateMilitaryHistory';
+
+describe('UpdateMilitaryHistory', () => {
+  let wrapper;
+  let oldSessionStorage;
+  let storage = {};
+  const servicePeriods = () => [
+    {
+      serviceBranch: 'Army',
+      dateRange: {
+        from: '2015-12-31',
+        to: '2020-12-31',
+      },
+    },
+  ];
+  const form = {
+    data: {
+      serviceInformation: {
+        servicePeriods: servicePeriods(),
+      },
+    },
+  };
+
+  function setUp(separationDate, test) {
+    const setFormData = data => {
+      form.data = data;
+    };
+    oldSessionStorage = window.sessionStorage;
+    delete window.sessionStorage;
+    window.sessionStorage = {
+      getItem: key => storage[key] || null,
+      setItem: (key, value) => {
+        storage[key] = value;
+      },
+      removeItem: key => delete storage[key],
+    };
+    if (separationDate) {
+      window.sessionStorage.setItem(SAVED_SEPARATION_DATE, separationDate);
+    }
+    wrapper = mount(
+      <UpdateMilitaryHistory form={form} setFormData={setFormData} />,
+    );
+    test();
+  }
+
+  afterEach(() => {
+    window.sessionStorage = oldSessionStorage;
+    storage = {};
+    wrapper.unmount();
+    form.data.serviceInformation.servicePeriods = servicePeriods();
+  });
+
+  it('should get called', () => {
+    setUp('', () => {
```

Comment:
NIT: Using positional arguments in the `setUp` function, and then passing `''` as the first argument initially made me think the first argument was accidentally blank. 

I wonder if that confusion could have been avoided by using a temporary variable or keyword arguments.

For example:
```
// temp variable
const separationDate = ''
setUp(separationDate, () => {...})

// keyword args
function setUp({ separationDate, test }) { ... }

setUp({ separationDate: '', test: () => { ... } })
```

Reply:
What if I passed in `null`?